examples/TestEm3/electrons.cu

```diff
@@ -80,8 +88,7 @@ static __device__ __forceinline__ void TransportElectrons(Track *electrons, cons
     for (int ip = 0; ip < 3; ++ip) {
       double numIALeft = currentTrack.numIALeft[ip];
       if (numIALeft <= 0) {
-        numIALeft                  = -std::log(currentTrack.Uniform());
-        currentTrack.numIALeft[ip] = numIALeft;
+        numIALeft = -std::log(currentTrack.Uniform());
```

Comment:
The write to `currentTrack.numIALeft[ip]` should not be necessary, because it is overwritten later

Reply:
Correct.